lib/lita/services/spreadsheet_manager.rb

```diff
@@ -3,7 +3,7 @@
 module Lita
   module Services
     class SpreadsheetManager
-      def initialize(ws_title)
+      def initialize(spreadsheet_key,ws_title)
```

Comment:
Unused method argument - spreadsheet_key. If it's necessary, use _ or _spreadsheet_key as an argument name to indicate that it won't be used.<br>Space missing after comma.